packages/hash/frontend/src/pages/[shortname]/entities/[entity-uuid].page/entity-editor/types-section/type-card.tsx

```diff
@@ -0,0 +1,81 @@
+import { faArrowsRotate, faAsterisk } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon, IconButton } from "@hashintel/hash-design-system";
+import { Box, Tooltip, Typography } from "@mui/material";
+
+import { WhiteCard } from "../../../../shared/white-card";
+
+interface TypeCardProps {
+  url: string;
```

Comment:
What URL is this? `BaseUri`? 

I'd also type this better, there are `VersionedUri` and `BaseUri` types we should be trying to use.

Reply:
It's just a url to pass to `href` for now. Not related with our branded types.